core/include/vecmem/memory/impl/device_atomic_ref.ipp

```diff
@@ -7,6 +7,15 @@
  */
 #pragma once
 
+#include <cassert>
+
+// MSVC include
+#if defined(_MSC_VER)
+// Not how you're supposed to do it, but whatever.
+#include <windows.h>
+#include <winnt.h>
+#endif
```

Comment:
I'm also not a fan of this. :frowning: Why not try to use the MSVC compiler intrinsics instead? :thinking: That's also not great of course, but with this we include the whole universe on Windows. :frowning: